src/unitxt/renderers.py

```diff
@@ -34,50 +34,26 @@ def process(
         return instance
 
 
-class RenderInstruction(Renderer, StreamInstanceOperator):
-    instruction: Instruction
-
-    def process(
-        self, instance: Dict[str, Any], stream_name: Optional[str] = None
-    ) -> Dict[str, Any]:
-        if self.instruction is not None:
-            instance["instruction"] = self.instruction()
-        else:
-            instance["instruction"] = ""
-        return instance
-
-
-class RenderFormat(Renderer, StreamInstanceOperator):
-    format: Format
-    demos_field: str = None
-
-    def process(
-        self, instance: Dict[str, Any], stream_name: Optional[str] = None
-    ) -> Dict[str, Any]:
-        demos_instances = instance.pop(self.demos_field, None)
-        if demos_instances is not None:
-            instance["source"] = self.format.format(
-                instance, demos_instances=demos_instances
-            )
-        else:
-            instance["source"] = self.format.format(instance)
-        return instance
-
-
 class StandardRenderer(Renderer, SequentialOperator):
     template: Template
     instruction: Instruction = None
     demos_field: str = None
-    format: ICLFormat = None
+    format: ModelInputFormatter = None
 
     steps: List[Operator] = InternalField(default_factory=list)
 
     def prepare(self):
         self.steps = [
             self.template,
             RenderDemonstrations(template=self.template, demos_field=self.demos_field),
-            RenderInstruction(instruction=self.instruction),
-            RenderFormat(format=self.format, demos_field=self.demos_field),
+            AddFields(
```

Comment:
Why set the instruction here on the instance, and not have it as a parameter to ModelInputFormatter?

Reply:
@yoavkatz , I will gladly remove `StandardRenderer`, along with `ICLFormat`. @elronbandel told me that the mission is to clean from renderers, and replace them by operators.